pages/create-release.js

```diff
@@ -0,0 +1,155 @@
+import { useState, useEffect } from 'react'
+import { supabase } from '@/utils/supabase'
+import { useUser } from '@/utils/context/user'
+
+const trueFalse = [
+  { id: 1, text: 'True', value: true },
+  { id: 2, text: 'False', value: false }
+]
```

Comment:
This feels strange, naming is very cryptic

Reply:
I agree that it feels strange, I think I can simplify this to not need the variable and still work. I have run into issues in the past while trying to use selects and set default states, and this way always worked the best. But it seems not right to use it for just true or false.